project_app/components/Header.js

```diff
@@ -0,0 +1,55 @@
+import styled from "styled-components";
+
+export default function Header() {
+  return (
+    <Container>
+      <header>
+        <h1>AIN'T BOARD</h1>
+        <div>
+          <Button>Search Board Game</Button>
+          <Button>Login</Button>
+          <Button>Join Us!</Button>
+        </div>
+      </header>
+    </Container>
+  );
+}
+
+const Container = styled.header`
+  header {
+    background: #353535;
+    display: flex;
+    position: absolute;
+    width: 1440px;
+    height: 204px;
+  }
+  h1 {
+    position: absolute;
+    width: 227.77px;
+    height: 34.34px;
+    margin-right: 80px;
```

Comment:
hmm, why would have h1 have dimensions?

Reply:
I initially thought we were pulling the code from figma but now I know better 👍